### DISCUSSION TITLE
venus-cluster社区测试技术报告

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/venus#4866] [Ideas]
## 背景

venus-cluster在2022年4月底挑选了4位社区成员，对venus-cluster进行了为期7天的性能测试。社区成员使用自己以前原有算力方案的硬件，改用新的venus-cluster方案，对封装性能进行对比测试。

## 技术报告

技术报告不同于之前面向社区的测试报告，更偏重于封装管道的调配以及问题排摸，还有社区成员使用的worker配置文件的实战用例。希望对社区起到一定的参考和帮助。每位参与测试的成员报告如下：

[社区测试报告_dennis.docx](https://github.com/filecoin-project/venus/files/8644347/_dennis.docx)
[社区测试报告-hanxianzai.docx](https://github.com/filecoin-project/venus/files/8644348/-hanxianzai.docx)
[社区测试报告-Tyler.docx](https://github.com/filecoin-project/venus/files/8644349/-Tyler.docx)
[社区测试报告_ovy.docx](https://github.com/filecoin-project/venus/files/8644346/_ovy.docx)

## 感谢社区

最后感谢每位参与测试的社区成员的努力和付出！venus-cluster也将再接再厉，不断优化迭代，期待更美好的明天！

**Comment by rickiey** [comment]
这些都是拿32G扇区测的吗？报告中都没有扇区信息

**Reply by Fatman13** [reply]
每个报告中，对应的worker配置文件中可以查询到扇区信息。
![image](https://user-images.githubusercontent.com/1591330/175447297-c32d34f3-355c-47ec-ac16-0f679c0b008b.png)
